spec/rspec/matchers/expecteds_for_multiple_diffs_spec.rb

```diff
@@ -0,0 +1,82 @@
+module RSpec
+  module Matchers
+    RSpec.describe ExpectedsForMultipleDiffs do
+
+      # Setup fake differ
+      let(:differ_factory) { double("Differ factory method", :call => differ) }
+      let(:differ) { double("Differ") }
+
+      before do
+        allow(differ).to receive(:diff) do |_actual, expected|
+          [_actual, expected].inspect
+        end
```

Comment:
I actually thought about that after I've written this. Since you have suggested it too, I will go for that.